src/views/studio/SignInView/SignInView.tsx

```diff
@@ -1,100 +1,69 @@
 import React from 'react'
-import { useNavigate } from 'react-router'
 
-import { BasicMembershipFieldsFragment } from '@/api/queries'
-import { SignInStepsStepper } from '@/components/SignInSteps'
-import { SvgActionNewChannel } from '@/components/_icons'
+import { Text } from '@/components/Text'
+import { Button } from '@/components/_buttons/Button'
+import { SvgActionChannel, SvgActionChevronL, SvgActionInformative } from '@/components/_icons'
+import { SvgJoystreamLogoFull } from '@/components/_illustrations'
 import { absoluteRoutes } from '@/config/routes'
-import { useConnectionStatusStore } from '@/providers/connectionStatus'
 import { useUser } from '@/providers/user'
 
 import {
-  CardWrapper,
-  HandleText,
+  BackLink,
+  ButtonGroup,
   Header,
-  Hero,
-  MemberGrid,
-  StyledAvatar,
-  StyledButton,
+  LogoContainer,
+  SignInButton,
+  StyledContainer,
+  StyledHero,
+  StyledSignInIllustrationSVG,
   SubTitle,
-  Wrapper,
 } from './SignInView.styles'
 
-export const SignInView = () => {
-  const navigate = useNavigate()
-  const { activeChannelId, setActiveUser, memberships } = useUser()
-  const nodeConnectionStatus = useConnectionStatusStore((state) => state.nodeConnectionStatus)
-  const internetConnectionStatus = useConnectionStatusStore((state) => state.internetConnectionStatus)
-
-  const handlePickMembership = async (membership: BasicMembershipFieldsFragment) => {
-    const newActiveUser = {
-      accountId: membership.controllerAccount,
-      memberId: membership.id,
-      channelId: activeChannelId,
-    }
+export type Membership = {
+  id: string
+  handle: string
+  about?: string
+  avatarUri?: string
+}
 
-    if (membership.channels.length) {
-      if (!activeChannelId) {
-        newActiveUser.channelId = membership.channels[0].id
-      }
-      setActiveUser(newActiveUser)
-      navigate(absoluteRoutes.studio.videos())
-    } else {
-      setActiveUser(newActiveUser)
-      navigate(absoluteRoutes.studio.newChannel())
-    }
-  }
+export const SignInView: React.FC = () => {
```

Comment:
Could we maybe rename it to `StudioWelcomeView` to make it more clear what's the point of this view?